dot/sync/message_test.go

```diff
@@ -49,7 +49,7 @@ func TestService_CreateBlockResponse_MaxSize(t *testing.T) {
 	addTestBlocksToState(t, maxResponseSize*2, s.blockState)
 
 	// test ascending
-	start, err := variadic.NewUint64OrHash(uint64(1))
+	start, err := variadic.NewUint32OrHash(uint32(1))
```

Comment:
supernit I think having just `1` instead of `uint32(1)` works. No need to change though.

Reply:
updated